tauri/src/styles/dark.css

```
@@ -0,0 +1,103 @@
+.dark,
+.dark[data-theme='dark']
+.dark[data-theme='custom'],
```

ESLint

[error] Replace `⏎` with `·`

(format/prettier)